Sources/MapboxDirections/MapMatching/MapMatchingResponse.swift

```diff
@@ -4,15 +4,39 @@ import FoundationNetworking
 #endif
 import Turf
 
+/**
+ A `MapMatchingResponse` object is a structure that corresponds to a map matching response returned by the Mapbox Map Matching API.
+ */
 public struct MapMatchingResponse: ForeignMemberContainer {
+    /**
+     Unrecognized properties that are documented as "beta" properties in the Mapbox Map Matching API response.
+     */
     public var foreignMembers: JSONObject = [:]
     
+    /**
+     The raw HTTP response from the Map Matching API.
+     */
     public let httpResponse: HTTPURLResponse?
     
+    /**
+     An array of `Match` objects.
+     */
     public var matches : [Match]?
```

Comment:
I wonder under what circumstances the Map Matching API will return a response with `matchings` set to `null`, resulting in this property being set to `nil`. If we don’t know, then we can leave that case unmentioned.

Reply:
I'll check if that's possible and update accordingly.